app/views/home/update.php

```diff
@@ -1,136 +1,191 @@
-        <input type="hidden" name="kode" id="kode" value="<?= $data['obat']['kode'] ?>">
-        <div>
-          <label class="form-label" for="nama-generik">nama generik</label>
-          <input minlength="1" maxlength="255" type="text" name="nama-generik" id="nama-generik" class="form-control" placeholder="nama generik" value="<?= $data['obat']['nama_generik'] ?>" required>
-          <div class="invalid-feedback">
-            Wajib diisi
-          </div>
-        </div>
-        <div class="mt-2">
-          <label class="form-label" for="nama-merek">nama merek</label>
-          <input minlength="1" maxlength="255" type="text" name="nama-merek" id="nama-merek" class="form-control" placeholder="nama merek" value="<?= $data['obat']['nama_merek'] ?>" required>
-          <div class="invalid-feedback">
-            Wajib diisi
-          </div>
-        </div>
-        <div class="mt-2">
-          <label class="form-label" for="deskripsi">deskripsi</label>
-          <textarea maxlength="65535" name="deskripsi" id="deskripsi" class="form-control" placeholder="deskripsi" required><?= $data['obat']['deskripsi'] ?></textarea>
-          <div class="invalid-feedback">
-            Wajib diisi
-          </div>
-        </div>
-        <div class="mt-2">
-          <label class="form-label" for="dosis">stok</label>
-          <div class="input-group">
-            <input type="number" name="stok" id="stok" class="form-control" min="0" max="2147483647" placeholder="stok" value="<?= $data['obat']['stok'] ?>" required>
-            <select class='form-select input-group' name='unit' id='unit'>
-              <?php
+    <div class="row">
+        <div class="col-lg-6">
+            <h2 class="text-capitalize">update obat</h2>
+            <a href="<?= BASEURL ?>/home" class=""><i class="bi bi-chevron-left"></i> kembali</a>
+            <form class="mt-2" action="<?= BASEURL ?>/home/ubah" method="post" novalidate>
+                <input type="hidden" name="kode" id="kode" value="<?= $data['obat']['kode'] ?>">
+                <div>
+                    <label class="form-label" for="nama-generik">nama generik</label>
+                    <input minlength="1" maxlength="255" type="text" name="nama_generik" id="nama_generik"
+                        class="<?=$data['format']['nama_generik']?>" placeholder="nama generik"
+                        value="<?= $data['obat']['nama_generik'] ?>" required>
+                    <?php if(!empty($data['errors']['nama_generik'])) { ?>
+                    <div class="invalid-feedback dblock">
+                        <?php echo $data['errors']['nama_generik']; ?>
+                    </div>
+                    <?php } ?>
+
+                </div>
+                <div class="mt-2">
+                    <label class="form-label" for="nama-merek">nama merek</label>
+                    <input minlength="1" maxlength="255" type="text" name="nama_merek" id="nama_merek"
```

Comment:
`for` di label tidak sesuai dengan `id` di input.
input tidak bisa fokus ketika klik label